logging/log_config.go

```diff
@@ -43,7 +44,7 @@ func buildRegexFromPattern(pattern string) string {
 	for _, ch := range pattern {
 		switch ch {
 		case '*':
-			matcher.WriteString(validLoggerSections)
+			matcher.WriteString(`.*`)
```

Comment:
I don't understand why this was `validLoggerSections`; `*` in a `log` pattern should be translated to `.*`. The interns seemed to agree before they left that this was a mistake.